cache-ehcache/src/test/groovy/io/micronaut/cache/ehcache/EhcacheCacheManagerFactorySpec.groovy

```diff
@@ -43,13 +43,18 @@ class EhcacheCacheManagerFactorySpec extends Specification {
         ApplicationContext ctx = ApplicationContext.run(["ehcache.caches.foo.enabled": true])
 
         when:
-        DefaultStatisticsService statisticsService = (DefaultStatisticsService) ctx.getBean(StatisticsService)
+        def statisticsService = ctx.getBean(StatisticsService)
         ctx.getBean(io.micronaut.cache.CacheManager) //Triggering CacheManager initialisation
 
+        then:
+        statisticsService.getCacheStatistics("foo")
+
+        when:
+        statisticsService.getCacheStatistics("not-foo")
 
         then:
-        statisticsService
-        statisticsService.started
+        def ex = thrown(IllegalArgumentException)
+        ex.message == "Unknown cache: not-foo"
```

Comment:
There's no `started` anymore, so I check for existence of a known cache, and that an unknown cache is missing